hack/dockerfile/install/gojunitreport.installer

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+GOJUNITREPORT_COMMIT=af01ea7f8024089b458d804d5cdf190f962a9a0c
+
+install_gojunitreport() {
```

Comment:
Are there advantages to using this instead of gotestsum? https://github.com/docker/cli/pull/1639

Reply:
i'll have a look. i started with go-junit-report because that's what i've used in the past and was easiest to setup.

Reply:
i decided to use `gotestsum` after getting to know it because it's better